custom/icds_reports/tasks.py

```diff
@@ -495,8 +495,9 @@ def _icds_add_awcs_to_file(csv_writer, error_type, rows):
 def _update_ucr_table_mapping():
     celery_task_logger.info("Started updating ucr_table_name_mapping table")
     for table in UCR_TABLE_NAME_MAPPING:
```

Comment:
I think that you want to delete these before creating them. table_type isn't a unique key, so it's possible that there will be multiple  for each if they are changed later (which they shouldn't be)